src/OpenTelemetry.Instrumentation.Process/MeterProviderBuilderExtensions.cs

```diff
@@ -26,17 +26,17 @@ namespace OpenTelemetry.Metrics;
 public static class MeterProviderBuilderExtensions
 {
     /// <summary>
-    /// Enables runtime instrumentation.
+    /// Enables process instrumentation.
     /// </summary>
     /// <param name="builder"><see cref="MeterProviderBuilder"/> being configured.</param>
     /// <returns>The instance of <see cref="MeterProviderBuilder"/> to chain the calls.</returns>
     public static MeterProviderBuilder AddProcessInstrumentation(this MeterProviderBuilder builder) => AddProcessInstrumentation(builder, configure: null);
 
     /// <summary>
-    /// Enables runtime instrumentation.
+    /// Enables process instrumentation.
     /// </summary>
     /// <param name="builder"><see cref="MeterProviderBuilder"/> being configured.</param>
-    /// <param name="configure">Runtime metrics options.</param>
+    /// <param name="configure">Process metrics options.</param>
```

Comment:
this method can be removed for now as there is nothing to configure in ProcessInstrumentaionOptions

Reply:
removed

Reply:
The option class seems to be also not needed.

Reply:
updated to use internal